Application/EdFi.Ods.Api/Controllers/DataManagementControllerBase.cs

```diff
@@ -59,7 +59,8 @@ public abstract class DataManagementControllerBase<TResourceReadModel, TResource
         private readonly IRESTErrorProvider _restErrorProvider;
         private readonly int _defaultPageLimitSize;
         private readonly bool _useProxyHeaders;
-
+        private readonly string _defaultForwardingHostServer;
```

Comment:
Why were these 2 variables introduced? seems that they aren't used

Reply:
Apparently it was a dead end. I should be able to remove it - either that, or I forgot to use these in an obscure code path.

Reply:
Removed